services/main-frontend/src/pages/manage/chatbots/[id]/[...path].tsx

```diff
@@ -0,0 +1,97 @@
+import { useQuery } from "@tanstack/react-query"
+import { useRouter } from "next/router"
+import { useTranslation } from "react-i18next"
+
+import MainFrontendBreadCrumbs from "@/components/MainFrontendBreadCrumbs"
+import ChatbotConfigurationForm from "@/components/page-specific/manage/courses/id/chatbot/ChatbotConfigurationForm"
+import {
+  configureChatbot,
+  deleteChatbot,
+  getChatbotConfiguration,
+} from "@/services/backend/chatbots"
+import { ChatbotConfiguration, NewChatbotConf } from "@/shared-module/common/bindings"
+import Button from "@/shared-module/common/components/Button"
+import ErrorBanner from "@/shared-module/common/components/ErrorBanner"
+import Spinner from "@/shared-module/common/components/Spinner"
+import { withSignedIn } from "@/shared-module/common/contexts/LoginStateContext"
+import useToastMutation from "@/shared-module/common/hooks/useToastMutation"
+import { assertNotNullOrUndefined } from "@/shared-module/common/utils/nullability"
+import { courseChatbotSettingsRoute } from "@/shared-module/common/utils/routes"
+import withErrorBoundary from "@/shared-module/common/utils/withErrorBoundary"
+
+const CustomizeChatbotPage = () => {
+  const { t } = useTranslation()
+  const router = useRouter()
+  const { id } = router.query
+
+  const chatbotQuery = useQuery<ChatbotConfiguration>({
+    queryKey: [`chatbot`, id],
+    queryFn: () => getChatbotConfiguration(assertNotNullOrUndefined(id?.toString())),
+    enabled: !!id,
+  })
+
+  const configureChatbotMutation = useToastMutation(
```

Comment:
Should we move the mutations to the form component? That way it would be easier to see when the mutations are pending there, and there would be less stuff to pass to the component.